tabbycat/api/serializers.py

```diff
@@ -105,3 +117,67 @@ def update(self, instance, validated_data):
         else:
             self.instance.speaker_set.set(speakers)
         return self.instance
+
+
+class SpeakerSerializer(serializers.ModelSerializer):
+    url = SpeakerHyperlinkedIdentityField(view_name='api-speaker-detail')
+    categories = TournamentHyperlinkedRelatedField(
+        many=True,
+        view_name='api-speakercategory-detail',
+    )
+
+    class Meta:
+        model = Speaker
+        fields = ('url','id','name', 'gender','email','phone','anonymous','pronoun',
```

Comment:
We normally try to keep spaces after commas. I don't know why it's disabled in flake8 at the moment, sorry, our bad. I just wanted to let you know that I'm about to reenable it.

Will look through the rest later, but at first glance this looks great! Thanks heaps for helping finally make this happen!